app/src/examples/PerfomanceMonitorExample.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useRef, useState } from 'react';
+import { Text, StyleSheet, Pressable, View } from 'react-native';
+import { PerformanceMonitor } from 'react-native-reanimated';
+
+import EmptyExample from './EmptyExample';
+import BokehExample from './BokehExample';
+import PlanetsExample from './PlanetsExample';
+import EmojiWaterfallExample from './EmojiWaterfallExample';
+
+enum Examples {
+  Empty = 'Empty Example',
+  Bokeh = 'Bokeh Example',
+  Planets = 'Planets Example',
+  Emojis = 'Emoji Waterfall Example',
+}
+
+export default function PerformanceMonitorExample() {
+  const exampleElements = useRef(
+    new Map<Examples, JSX.Element>([
+      [Examples.Empty, <EmptyExample />],
+      [Examples.Bokeh, <BokehExample />],
+      [Examples.Planets, <PlanetsExample />],
+      [Examples.Emojis, <EmojiWaterfallExample />],
+    ])
+  );
+
+  const [currentExample, setCurrentExample] = useState(<EmptyExample />);
```

Comment:
It is generally not suggested to store as heavy objects as entire components in `useState` hook. I would suggest to store `Examples` enum values in there: 
```suggestion
  const [currentExample, setCurrentExample] = useState<Examples>(Examples.Empty);
```